daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -51,11 +51,20 @@ archive c = exercise c Archive
 stakeholder : Template c => c -> [Party]
 stakeholder c = signatory c ++ observer c
 
--- | HIDE pass the @c@ using type applications, otherwise it's ambiguous
+-- | Look up the contract id associated with a given contract key.
```

Comment:
I'd usually uppercase ID (here and below) but up to you
```suggestion
-- | Look up the contract ID associated with a given contract key.
```

Reply:
The surrounding documentation uses "contract id" rather than "contract ID". I'd like to blend in.